contrib/IECoreAppleseed/src/IECoreAppleseed/RendererImplementation.cpp

```diff
@@ -924,10 +924,10 @@ void IECoreAppleseed::RendererImplementation::setCamera( const string &name, Cam
 	// crop window
 	const Box2fData *cropWindow = cortexCamera->parametersData()->member<Box2fData>( "cropWindow" );
 	asf::AABB2u crop;
-	crop.min[0] = cropWindow->readable().min.x * ( res[0] - 1 );
-	crop.min[1] = cropWindow->readable().min.y * ( res[1] - 1 );
-	crop.max[0] = cropWindow->readable().max.x * ( res[0] - 1 );
-	crop.max[1] = cropWindow->readable().max.y * ( res[1] - 1 );
+	crop.min[0] = (int)cropWindow->readable().min.x * ( res[0] - 1 );
```

Comment:
Have you tested this, or do we have unit test coverage for it? I _think_ the cast to int binds to `cropWindow->readable().min.x` rather than the result of the whole calculation. Meaning it's going to either be 0 or 1 before the multiplication, meaning crop windows would be broken.